sei-devnet-2/genesis.json

```diff
@@ -0,0 +1,1370 @@
+{
+  "genesis_time": "2023-01-11T22:00:00.000000000Z",
```

Comment:
![image](https://user-images.githubusercontent.com/18161326/211668415-ce260c52-7a68-41ec-8f04-577cf6d49b96.png)

Reply:
Depending when we can distribute, might just move to Thursday 9am PST